blazor/appearance/themes.md

```diff
@@ -32,6 +32,24 @@ The following list of themes are included in the Syncfusion Blazor components li
 
 The Syncfusion Blazor Bootstrap Theme is designed based on Bootstrap v3, whereas the Bootsrap4 theme is designed based on Bootstrap v4.
 
+## Optimized CSS files
+
+Syncfusion controls provide two size modes for each theme: **normal** and **bigger**. By default, the **normal size mode** is applied, offering a standard appearance that is suitable for all devices. The **bigger size mode** provides an enlarged interface for enhanced interactions, visibility, and an overall improved user experience by increasing the size of the UI controls.
+
+The bigger size mode is ideal when an enlarged appearance is needed, while the normal size mode works best for maintaining the default appearance of controls.
+
+Below is a comparison of the Button control in normal and bigger size modes:
+
+![bigger-sized-button](./images/bigger-theme-button.png)
+
+Each theme includes both normal and bigger size modes, which increases the overall file size. To optimize performance, Syncfusion offers additional theme files (e.g., `fluent2-lite.css`), which only include the normal size mode styles. This results in a significant reduction in file size, improving load times and overall application performance, especially when the bigger size mode is unnecessary.
+
+Refer to the comparison below for the default and optimized theme file sizes:
+
+| Theme Name | Default Theme Size | Optimized Theme Size |
+| -- | -- | -- |
+| Fluent 2 | 3.97 MB | 2.96 MB |
+
```

Comment:
we should specifiy all theme name with lite naming. Before that check in our live package for confirming all lite themes are there. @Athiswaran-SF3980

Reply:
added lite theme name in existing theme table